src/styles/blocks/card.scss

```diff
@@ -0,0 +1,93 @@
+@use "./stars.scss";
+
+@use "./utils/variables" as *;
+
+.card {
+  width: $card-width;
+  padding: 32px 16px 16px;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__photo {
+    display: flex;
+    object-fit: contain;
+    width: 100%;
+    height: 134px;
+    padding: 0 3px;
+    margin-bottom: 40px;
+  }
+
+  &__title {
+    padding-bottom: 4px;
+
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+  }
+
+  &__code {
+    margin-bottom: 16px;
+
+    font-size: 10px;
+    color: #616070;
+    line-height: 14px;
+  }
+
+  &__rating {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 24px;
+  }
+
+  &__count-reviews {
+    font-size: 10px;
+    line-height: 14px;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 16px;
+
+    line-height: 18px;
+  }
+
+  &__price-text {
+    color: #616070;
+  }
+
+  &__price-value {
+    font-weight: bold;
+    font-size: 16px;
+  }
+
+  &__button {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    padding: 10px 0;
+
+    font-weight: bold;
+    font-size: 14px;
+    text-transform: uppercase;
+    color: #fff;
+    text-decoration: none;
+    line-height: 18px;
+
+    background-color: $color-light-blue;
+    border-radius: 5px;
+    border: 1px solid $color-light-blue;
+
+    transition: background-color 0.5s,
+      color 0.5s;
```

Comment:
Would be nice to create a variable for duration, then we can use it for every transition to make it's duration consistent with other page transitions